internal/cmd/templates.go

```diff
@@ -161,6 +221,86 @@ var (
 	grpcCmdTemplate = &cmdTemplate{
 		Use:   "grpc dirOrProtoFiles...",
 		Short: "Call a gRPC endpoint. Be sure to set required flags address, method, and either data or stdin.",
+		Long: `What this does behind the scenes:
+
+- Compiles your Protobuf files with "protoc", generating a "FileDescriptorSet".
```

Comment:
I think this part may be unnecessary, describing a `FileDescriptorSet`. What if we simplified it as:

```
This command compiles your proto files with "protoc", converts JSON input to binary
and converts the result from binary to JSON. All these steps take on the order of 
milliseconds. For example, the overhead for a file with four dependencies is about
30ms, so there is little overhead for CLI calls to gRPC.
```

Commenting here to get both your thoughts, not sure if this is any clearer.